### DISCUSSION TITLE
Is this repository still maintained

### DISCUSSION THREAD

**Discussion by Shogobg** [Mermade/widdershins#523] [General]
Hello @MikeRalphson ,
I was wondering if you're still maintaining this repository, since it hasn't been updated in some time?

**Comment by MikeRalphson** [comment]
I do plan to get back to it and do another release sometime this year, but it's held up by some unfinished work in dependent projects and general lack of time.

**Reply by fzn0x** [reply]
> I do plan to get back to it and do another release sometime this year, but it's held up by some unfinished work in dependent projects and general lack of time.

Do you have any opencollective link? most people I know kinda not fit with github sponsors payment system
